services/ui-src/src/components/fields/DynamicField.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+//components
+import { DynamicField } from "components";
+
+const appendButtonText = "Add a row";
+
+jest.mock("react-hook-form", () => ({
+  useFormContext: () => ({
+    register: () => {},
+  }),
+  useFieldArray: jest.fn().mockReturnValue({
+    fields: [
+      {
+        id: "uniqueId",
+        0: {
+          0: "text",
+        },
+      },
+    ],
+    append: () => {},
+    remove: () => {},
```

Comment:
will want to make these `jest.fn()` so we can spy on them later in some new tests.